nixos/modules/services/web-apps/languagetool.nix

```diff
@@ -0,0 +1,47 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  cfg = config.services.languageToolHttp;
+  format = pkgs.formats.toml { };
+in
+{
+  options.services.languageToolHttp = {
+    enable = mkEnableOption "the LanguagTool http server";
```

Comment:
It seems relevant to include [this](https://dev.languagetool.org/http-server#using-ssltls) in the description here.